.github/workflows/sugar_unit_test.yaml

```diff
@@ -34,12 +34,24 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - uses: subosito/flutter-action@v2.18.0
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+      - name: Install Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "adopt"
+          java-version: "11.0.26+4"
```

Comment:
Is there a reason we need to build specifically against Java 11?

Reply:
You can pin it against any version you want.
Java 11 is used on every version of android (except the latest one) so I chose to use that one.

Reply:
Fair enough